### ISSUE TITLE
feat(draft-workspace): be able to massively add observables to a workbench (and then relationships)

### ISSUE THREAD

**Issue by SamuelHassine** [LuatixHQ/opencti#3189]
## Use case

Be able to massively add observables to a workbench.

1. Select a type
2. Copy/paste a list of values
3. In the Update section, be able to link all selected observables to a specific entity

**Comment by Jipegien** [comment]
We won't invest in Workbench, but definitely a feature for Draft Epic (#3195).

**Comment by ParamConstructor** [comment]
Possibly related? - https://github.com/OpenCTI-Platform/opencti/pull/6211

**Comment by Jipegien** [comment]
Yes it is. Partially at least, as the feature you developed does not cover all SCOs

**Comment by SamuelHassine** [comment]
🤖 [AI-generated]

**feat: be able to massively add observables to a workbench (and then relationships)**
→ **feat: allow bulk-adding observables to a workbench (and then relationships)**

"be able to massively add" is a modal/ability phrase — replaced with imperative verb "allow bulk-adding" per Conventional Commits convention. Label `feature` added (required for `feat:` type).
